Readme.md

```diff
@@ -0,0 +1,27 @@
+TYPING SPEED TEST PYTHON.
+
+
+open to all for contributions.
+all valid PRs will be merged .
+(beginner-friendly) and easy first contribution.
+
+a software to measure the typing speed of user.
+currently it takes sentences from only one file.
+
+Steps to contribute.
+
+1.Register your GitHub ID on Hackoberfest Website
+
+2.Fork this repository.
+
+3.Make the required Changes.(make sure there are no conflicts with existing codes).
```

Comment:
Tell the required commands 
Git clone git add git commit 
Like adding meaningful commit messages